### ISSUE TITLE
Room is printing as None in notification mail

### ISSUE THREAD

**Issue by ericmjonas** [uchicago-cs/cs121-officehours-webapp#18]
The notification mail I received said

> We wanted to let you know that we have scheduled you to attend office hours at 08:45 PM - 09:00 PM on Oct. 3, 2020. Please make sure to go to None at that time.

I believe the None is arising due to the `{{request.actual_slot.room}}` in the email template. 

Perhaps instead we could mail them the zoom link in case they forget it?

**Comment by semidirect** [comment]
The same issue occurs in the text that shows up on the /cmsc12100-aut-20/my-request page when the request is scheduled or in progress.

**Comment by borjasotomayor** [comment]
For the e-mail notification, how about the following?


> Greetings from the {{request.course_offering.catalog}} Office Hours!
> 
> We wanted to let you know that we have scheduled you to attend office hours at {{request.actual_slot.interval}} on {{request.actual_slot.date}}. You will be seen by {{request.server}}.
> 
> As a reminder, you provided the following Zoom URL: {{request.zoom_url}}
> 
> Please make sure this meeting is open during your scheduled time. An instructor/TA will join the meeting sometime during your scheduled slot.
> 
> If you are not planning to attend office hours after all, please cancel your request at https://cs121-officehours.cs.uchicago.edu/. Cancelling a request will not affect your standing in future requests and, once you cancel your request, you will be allowed to make another request for the same day.

**Comment by semidirect** [comment]
I'm not sure we want to promise who they will be seen by. I am imagining that if there are multiple instructors/TAs in a particular office hours slot, they'll probably mostly help the next person in the slot, even if it's not who scheduled them. There may be exceptions, like if a student was following up with the same TA for an earlier issue they had discussed, or if one TA feels less qualified to help with a problem than another, but also I remember last year the work of doing the scheduling might be more heavily done by some of the TAs/instructors.

Otherwise I think it looks good from the perspective of showing where they are meeting.

I think it would be useful to include the text of their request too (with an instruction to update the text if their request changes), but I can raise that as a separate issue.

**Comment by borjasotomayor** [comment]
I had added the "server name" mostly so students knew who to expect in their Zoom call, but I agree we probably want the flexibility for someone to schedule a student, and then hand them off to someone else (e.g., if they're still busy helping someone out). I think we just need to make sure the students know the instructors/TA's names, so they know who could potentially show up in their Zoom call.

**Comment by semidirect** [comment]
Good point that they need to know who is an acceptable person to join their call. It's probably okay if they just have a list of all the instructors/TAs, even without knowing specifically who is going to see them or the list of people on duty at the time.

**Comment by borjasotomayor** [comment]
The "None" on the web interface should be fixed now. Both changes (including the updated e-mail template) are on staging now.
